write-good.js

```diff
@@ -49,4 +55,29 @@ function dedup (suggestions) {
   }, []);
 }
 
+// Mozilla Object.Assign Polyfill
+if (typeof Object.assign != 'function') {
```

Comment:
This is likely a module somewhere. Use this? https://www.npmjs.com/package/object.assign